### PR TITLE
feat: publish to vsc marketplace

### PR DESCRIPTION
**Pull request by michawai**
Before submitting a pull request, please ensure the following:

## Description

Please explain the changes you made.

Add a link to the design (if applicable).

### Checklist

- [ ] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [ ] Linting run locally successfully
- [ ] All tests pass
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

**Review by allacherkes — CHANGES_REQUESTED** [review]

File: packages/tasks_panel/package.json [hunk 1]
```diff
@@ -108,9 +108,15 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "VSCode Tasks Panel",
+      "title": "Task Explorer",
       "properties": {
-        "VSCode Tasks Panel.loggingLevel": {
+        "TaskExplorer.enableSapWebAnalytics": {
```
[the remark this answers is not in the document]
Comment (michawai) [reply packages/tasks_panel/package.json]: Fixed

File: packages/tasks_panel/package.json [hunk 2]
```diff
@@ -108,9 +108,15 @@
...
+          "type": "boolean",
+          "default": true,
+          "description": "Enable collecting usage analytics data for Application Wizard.  If enabled, non-personally identifiable information is used to help understand the product usage and improve the tool.",
+          "scope": "resource"
+        },
+        "TaskExplorer.loggingLevel": {
```
[the remark this answers is not in the document]
Comment (michawai) [reply packages/tasks_panel/package.json]: Fixed

File: packages/tasks_panel/package.json [hunk 3]
```diff
@@ -125,7 +131,7 @@
           "description": "The verbosity of logging according to the following order: trace > debug > info > warn > error > fatal > off.",
           "scope": "resource"
         },
-        "VSCode Tasks Panel.sourceLocationTracking": {
+        "TaskExplorer.sourceLocationTracking": {
```
[the remark this answers is not in the document]
Comment (michawai) [reply packages/tasks_panel/package.json]: Fixed

File: packages/tasks_panel/package.json [hunk 4]
```diff
@@ -108,9 +108,15 @@
     ],
     "configuration": {
       "type": "object",
-      "title": "VSCode Tasks Panel",
+      "title": "Task Explorer",
       "properties": {
-        "VSCode Tasks Panel.loggingLevel": {
+        "Task Explorer.enableSapWebAnalytics": {
```
Comment (allacherkes) [inline packages/tasks_panel/package.json:113]: here should be package name according to documentation

**Review by allacherkes — CHANGES_REQUESTED** [review]

File: packages/tasks_panel/package.json (hunk 4 above)
Comment (michawai) [reply packages/tasks_panel/package.json]: updated to package name

**Pull request merged**